vendordeps/REVLib.json

```diff
@@ -0,0 +1,73 @@
+{
+    "fileName": "REVLib.json",

```

Comment:
Do you have any Rev products on your robot?

Reply:
gone, reduced to atoms D;;;;; up thanos butt